### PR TITLE
Remove step by step guide for simple commands

### PR DESCRIPTION
**Pull request by yeo-kexin**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -190,15 +185,15 @@ Format: `exit`
 
 #### Saving the data
 
-Projact data are saved in your hard disk automatically after any command that changes the data so there is no need to save manually. Don't worry, the data you have entered in previously will be automatically loaded when you reopen Projact the next time.
+Projact's data is saved to your hard disk automatically after any changes in the data so there is no need to save manually. Don't worry, the data you have entered in previously will be automatically loaded when you reopen Projact the next time.
```
Comment (JoeyToh) [inline docs/UserGuide.md]: "... after there are any changes in the data..."? Not sure about the sentence flow/grammar

**Review by JoeyToh — COMMENTED** [review]
lgtm other than the comment

**Review by siqiang-ng — APPROVED** [review]
LGTM after joey's comment part is edited.

File: docs/UserGuide.md (hunk 1 above)
Comment (yeo-kexin) [reply docs/UserGuide.md]: okie, thanks :D

**Pull request merged**